src/gpl/src/mbff.cpp

```diff
@@ -0,0 +1,1024 @@
+#include "mbff.h"
+#include <iostream>
+#include <algorithm>
+#include <queue>
+#include <stack>
+#include <map>
+#include <set>
+#include <ctime>
+#include <chrono>
+#include <memory>
+#include <random>
+#include <ortools/linear_solver/linear_solver.h>
+#include <ortools/base/logging.h>
+#include <ortools/sat/cp_model.h>
+#include <ortools/sat/cp_model.pb.h>
+#include <ortools/sat/cp_model_solver.h>
+#include <lemon/list_graph.h>
+#include <lemon/network_simplex.h>
+#include <lemon/maps.h>
+using namespace std;
+using namespace lemon;
+using namespace operations_research;
+using namespace sat;
```

Comment:
Remove `using namespace *;`. It's much better to leave the namespace alone. If you must shorten the namespace prefer to use `using ortools = operations_research` or something like that.